packages/dev-frontend/src/LiquityFrontend.tsx

```diff
@@ -38,7 +38,7 @@ const UnsupportedMainnetFallback: React.FC = () => (
     </Heading>
 
     <Paragraph sx={{ mb: 3 }}>
-      Please change your network to Ropsten.
+      Please change your network to Rinkeby.
```

Comment:
Rinkeby is going to shut down in Q4 2022 we should probably switch to Goerli

Reply:
Resolved